src/commcare_cloud/ansible/roles/nginx/tasks/install.yml

```diff
@@ -90,10 +90,33 @@
     owner: root
     group: root
   no_log: true
-  when: not fake_ssl_cert and nginx_key_value
+  when: not fake_ssl_cert and nginx_key_value | default(None)
   tags:
     - update-cert
 
+
+- name: Link CCHQ SSL cert
+  become: yes
+  file:
+    src: "/etc/letsencrypt/live/{{SITE_HOST}}/fullchain.pem"
+    dest: "{{ ssl_certs_dir }}/{{ nginx_ssl_cert }}"
+    state: link
+  when: not fake_ssl_cert and letsencrypt_cchq_ssl|default(False)
```

Comment:
I think `letsencrypt_cchq_ssl` will be set to `False` if it doesn't appear in `proxy.yml`, so no need to test if it's defined